gc/gc-base/src/main/java/org/projectnessie/versioned/persist/gc/ContentValues.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.versioned.persist.gc;
+
+import java.io.Serializable;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+import org.projectnessie.model.Content;
+import org.projectnessie.model.ContentKey;
+
+/** Contains details about one content id: key and the expired values. */
+public class ContentValues implements Serializable {
+
+  private static final long serialVersionUID = -1268447510930923804L;
+  private ContentKey contentKey;
```

Comment:
I too don't have a clarity at the moment. Once we finalize the consumer (phase2) design. we can change this if it requires. It will be a small change. I thought consumer cannot understand content id. So, I kept the content key.
For example Iceberg's expire snapshots need table name, not the content id. so `contentKey` is needed.

Reply:
Again: the table name is only valid at a specific commit.
You're unsure whether and how it will be useful - so it's better to remove it here and add it when it's needed.

Reply:
I think I need a map of ref + hash  --> key.

I will remove keys now. 

Later while working on ph2 I will add it with other required info.